### DISCUSSION TITLE
Why is my setting not giving data back?

### DISCUSSION THREAD

**Discussion by bwired-nl** [tbnobody/OpenDTU#202] [Q&A]
I have installed everything according to the manual. but not getting reply from my dtu-1500
I have a esp_32_devkit v4 with ICQUANZX NRF24L01+ 
I have used the pinsettings axeactly as on the drawing
Chnage the Pinsetting in the platform.io ini to below (strange is that in the firmware this was set to other ports??)

-DHOYMILES_PIN_MISO=19
-DHOYMILES_PIN_MOSI=23
-DHOYMILES_PIN_SCLK=18
-DHOYMILES_PIN_IRQ=16
-DHOYMILES_PIN_CE=4
-DHOYMILES_PIN_CS=5

But I get below, keeps polling and after 30 secs it stops
I have used all the differtenet powersettings
Put The board on a separte power adapter
Closeby the DTU
etc etc

1 extra question, How do I get the DTU serial??
its auto filled in, i think thats is OK

My Setting DTU is OK, If I use DTU-AHOY version this gives me the readings of the DTU and Panels.

Hope somebody can help me
Thanks!

Starting OpenDTU
Initialize FS... done
Reading configuration... done
Initialize Network... done
Setting Hostname... Configuring WiFi STA using new credentials... done
Initialize NTP... done
Initialize MqTT... done
Initialize WebApi... done
Initialize Hoymiles interface... Connection successfull
  Setting radio PA level... 
  Setting DTU serial... 
  Setting poll interval... 
  Adding inverter: 116181852181 - Hoymiles1 done
done
Switch to WiFi mode
Setting Hostname... done
WiFi connected
Configuring WiFi STA using existing credentials... done
Configuring WiFi STA DHCP IP... done
WiFi got ip: 10.0.0.145
Network connected
Connecting to MQTT...
Connected to MQTT.
Fetch inverter: 116181852181
Request SystemConfigPara
Fetch inverter: 116181852181
Request SystemConfigPara
Fetch inverter: 116181852181
Request SystemConfigPara

**Comment by tbnobody** [comment]
Hi,

If you don't see any messages like:
`TX RealTimeRunData Channel: 40 --> 15 71 60 35 46 78 56 34 12 80 0B 00 63 41 9E 79 00 00 00 00 00 00 00 00 69 1C 44 `

The problem is most likely that no current time is set. Maybe your ESP does not have a valid internet connection or is unable to reach the configured NTP server?

**Reply by bwired-nl** [reply]
NTP is Set, thats not the problem, time is synced!

**Reply by tbnobody** [reply]
Are you getting TX messages or not? In the serial log above you don't show any TX messages. 
Here (https://github.com/tbnobody/OpenDTU/issues/193#issuecomment-1272351721) you wrote that you are getting TX messages.
Whats is now correct?

Because if you are getting TX message then NTP/Clock setup is correct. If you don't get TX message, the clock is not set.

**Comment by bwired-nl** [comment]
yes i get, see below, thanks!

Starting OpenDTU
Initialize FS... done
Reading configuration... done
Initialize Network... done
Setting Hostname... Configuring WiFi STA using new credentials... done
Initialize NTP... done
Initialize MqTT... done
Initialize WebApi... done
Initialize Hoymiles interface... Connection successfull
Setting radio PA level...
Setting DTU serial...
Setting poll interval...
Adding inverter: 116181852181 - Hoymiles1 done
done
Switch to WiFi mode
WiFi connected
Setting Hostname... done
Configuring WiFi STA using existing credentials... done
Configuring WiFi STA DHCP IP... done
WiFi got ip: 10.0.0.145
Network connected
Connecting to MQTT...
Connected to MQTT.
Fetch inverter: 116181852181
Request SystemConfigPara
TX RealTimeRunData Channel: 23 --> 15 81 85 21 81 78 56 34 12 80 0B 00 63 42 88 7D 00 00 00 00 00 00 00 00 67 C0 41
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 40 --> 15 81 85 21 81 78 56 34 12 80 0B 00 63 42 88 7D 00 00 00 00 00 00 00 00 67 C0 41
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 61 --> 15 81 85 21 81 78 56 34 12 80 0B 00 63 42 88 7D 00 00 00 00 00 00 00 00 67 C0 41
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 75 --> 15 81 85 21 81 78 56 34 12 80 0B 00 63 42 88 7D 00 00 00 00 00 00 00 00 67 C0 41
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 3 --> 15 81 85 21 81 78 56 34 12 80 0B 00 63 42 88 7D 00 00 00 00 00 00 00 00 67 C0 41
RX Period End
All missing
